### PR TITLE
Add rapidjson include headers for test

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

File: src/test/CMakeLists.txt [hunk 1]
```diff
@@ -37,4 +37,7 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(googletest)
 
-add_subdirectory(triton_json triton_json)
+if (TRITON_COMMON_ENABLE_JSON)
+    add_subdirectory(triton_json triton_json)
```
Comment (mc-nv) [inline src/test/CMakeLists.txt:41]: I suppose we need to change option here to be able use this condition.
https://github.com/triton-inference-server/common/blob/main/CMakeLists.txt#L40

Comment (GuanLuo) [reply src/test/CMakeLists.txt:41]: I wonder if that has been turned off explicitly in FIL backend build.

Comment (Tabrizian) [reply src/test/CMakeLists.txt:41]: This was on in the FIL backend. Not sure why it was only failing in case of FIL backend.

Comment (Tabrizian) [reply src/test/CMakeLists.txt:41]: @mc-nv This is a parameter that the user can set ON/OFF if it needs TritonJson or not.

**Pull request merged**